### DISCUSSION TITLE
英文单词之间的空格识别效果差

### DISCUSSION THREAD

**Discussion by swwsiwwe** [PaddlePaddle/PaddleOCR#13870] [Q&A]
我使用OCRv4转为onnx模型，在对网上的英文论文进行截图识别，截图中的英文句子基本占满图片，使用模型发现每个字母标点能够识别，但是单词之间的空格识别很差，字母基本连在一块。当开启use_dilation时效果会稍微好一点。
        把图片拼接几块空白后效果稍微变好。
        把图片放大后效果很好。
        在word上测试字号，发现当字号很小 "< 11"时， “times new roman”字体比起“西文正文”的英文空格更容易识别到。
        如何提高模型对小字号英文空格识别的准确性呢？

**Comment by UserWangZz** [comment]
目前en_ppocrv4的识别结果容易出现括号，标点，空格的识别不到的情况。未来将会开源en_PPOCRv4_server模型，能够比较有效改善这类问题

**Reply by OeasyC** [reply]
只识别大小写字母和数字，加载v4 pretrain训练大概需要多少样本。场景很单一。 😊

**Reply by UserWangZz** [reply]
这个开源的v4模型不能满足需求吗？

**Reply by OeasyC** [reply]
> 这个开源的v4模型不能满足需求吗？
应用场景是工业方面，显示屏喷墨的字符，点状。

**Reply by UserWangZz** [reply]
那应该200-500张图片就可以了
